pkg/plugin/plugin.go

```diff
@@ -366,6 +383,19 @@ func CmdAdd(args *skel.CmdArgs) error {
 		}
 
 		err = contNetns.Do(func(_ ns.NetNS) error {
+			for _, ipc := range newResult.IPs {
+				containerMac := IPAddrToHWAddr(ipc.Address.IP)
+				containerLink, err := netlink.LinkByName(args.IfName)
+				if err != nil {
+					return fmt.Errorf("failed to lookup container interface %q: %v", args.IfName, err)
+				}
+				err = assignMacToLink(containerLink, containerMac, args.IfName)
```

Comment:
What if the user passes a MAC explicitly through `requestedMac` https://github.com/k8snetworkplumbingwg/ovs-cni/pull/205/files#diff-fc137d62147eb6a9c89c0566479ff219320ca691f2505631ebacbe513bcc8108R136, this would override it, right? I think we should use the IP based MAC only as a fallback.

Reply:
@phoracek done.